bouncer/commands/create_lp_pool.ts

```diff
@@ -6,12 +6,12 @@
 // initial price in USDC
 // For example: ./commands/create_lp_pool.ts btc 10000
 
-import { Asset } from '@chainflip/cli';
+import { InternalAsset as Asset } from '@chainflip/cli';
 import { createLpPool } from '../shared/create_lp_pool';
 import { runWithTimeout } from '../shared/utils';
 
 async function main() {
-  const ccy = process.argv[2].toUpperCase() as Asset;
+  const ccy = process.argv[2] as Asset;
```

Comment:
we loose the input being case insensitive for the commands. we could fix that by comparing the lowercase version of the input the the lowercase version of all supported assets, but that means maintaining a list of supported assets. so no strong opinion

Reply:
okay looks like there already is a `allInternalAssets` in `bouncer/shared/swapping.ts`, so could move that into `bouncer/shared/utils.ts`. but still no strong opinion

Reply:
I thought about that too but I'm not sure there is a nice solution besides, as you say, maintain a list of supported assets that convert from lower/upper case to camel case assets (`InternalAssets`). The `allInternalAssets` in `bouncer/shared/swapping.ts` is not that list, it's a temporal workaround to extend `InternalAssets` with `ArbEth` and `ArbUsdc`, so it's useless there.